jekyll-assets/_layouts/boxes.html

```diff
@@ -1,19 +1,20 @@
 <!doctype html>
 <html lang="en">
   <head>
-    {% include head.html %} 
-    <link rel="stylesheet" href="{{ site.baseurl }}/css/boxes.css?ver={{ site.time | date: '%s' }}">
+    {% include head.html %}
   </head>
-  <body>
-    {% include header.html %}      
-    {% if page.name == "404.adoc" %}
-      {% include toptitle404.html %}
-      {% include subtitle404.html %}
-    {% else %}
-      {% include toptitle.html %}
-      {% include subtitle.html %}
-    {% endif %}
-    {% include tabs.html %}
+  <body onload="makeToc()">
```

Comment:
As above re being unobtrusive: could you fire `makeToc()` on `DOMContentLoaded`?

```javascript
document.addEventListener("DOMContentLoaded", () => {
  makeToc();
});
```

Reply:
Similar question as the toggle comment above -- happy to move the call, but I'd like to know if there's a performance or organisational benefit before I do that!

Reply:
If you can extract all your JavaScript into a separate file then it can be cached across page loads and served from our CDN while the HTML is typically always loaded from the origin servers.

Reply:
Ah, gotcha. I can pull the makeToc and theme stuff into a separate file, but the inline toc manipulation will have to stay here as it's part of the static site generation process.